projects/igniteui-angular/src/lib/core/styles/components/dialog/_dialog-theme.scss

```diff
@@ -38,7 +38,8 @@
     $background: null,
     $title-color: null,
     $message-color: null,
-    $shadow: null
+    $shadow: null,
+    $bootstrap-border-color: null
```

Comment:
We shouldn't add theme specific properties to otherwise variant-agnostic functions.